### DISCUSSION TITLE
Minutes and months are both denoted by "m"

### DISCUSSION THREAD

**Discussion by introt** [community/community#57730] [Discussions]
### Select Topic Area

Bug

### Body

See title. As can be seen in the attached screenshot, one can not discern minutes from months. Please use eg. "min" for minutes.

![image](https://github.com/community/community/assets/26069787/9e55ebfb-c90a-4a71-96cb-1880dc72cf7c)

**Comment by LindseyB** [comment]
I've created a bug report with the team. Thanks for the report.
